src/css/book.css

```diff
@@ -0,0 +1,209 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Slab:wght@300;700&display=swap');
+
+.bookCard {
+  width: 80%;
+  height: 10.625rem;
+  margin: 2.313rem auto;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.category {
+  width: 3.063rem;
+  height: 1.125rem;
+  margin: 0 4rem 0 0;
+  opacity: 0.5;
+  font-size: 0.875rem;
+  font-family: 'Montserrat', sans-serif;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
```

Comment:
- [ ] I would recommend taking a look through all the CSS files and if you find more issues like this create a class for that repeated property and use it in the respective elements. 😸
    - Please check the examples above to get an idea of what you can do to solve this issue 🚀